packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx

```diff
@@ -44,22 +49,39 @@ const StyledScrollableNav = styled.div`
       pointer-events: none;
       background: linear-gradient(
         ${({ dir }) => (dir === 'ltr' ? 'to right' : 'to left')},
-        ${C_POSTBOX_TRANSPARENT} 0%,
-        ${C_POSTBOX_OPAQUE} 100%
```

Comment:
Where are these original values set after this change, I'd hope they would be defaults that can be overriden for sport etc?

Reply:
Yeah, these use the values passed in as props now, and add the transparency to them.

Reply:
So there's not a default, per the issue the colours become required props

Reply:
Okay, but where the prop is being used, is there still a default value that is used in the prop isn't provided? It would make this a 'non-breaking' change on the Simorgh side potentially. Or I suppose you will integrate this with a PR that includes all these values in the service config?

Reply:
My intent was to add all the values to the service configs, yes